src/test/kotlin/com/atlassian/performance/tools/ssh/api/SshTest.kt

```diff
@@ -63,6 +64,29 @@ class SshTest {
         }
     }
 
+    @Test
+    fun shouldTimeOutWhenStopping() {
+        val uninterruptibleSleep = """
+            trap 'echo "got interrupted"' INT
+            for i in {1..10}
+            do
+                echo "sleeping ${'$'}i..."
```

Comment:
I don't understand this substitution, what is it supposed to do?

Reply:
`${'$'}` is a way to print a literal `$` within a multiline string literal `"""`.

Reply:
I tested it on Bash and ZSH and in both cases I got "bad substitution" error.

Reply:
OMG, brain fog. It' a Kotlin substitution 🤦 . Ignore me :)

Reply:
If you want to restore your bash session to normal, you can clear the trap via `trap - INT`